src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -603,7 +603,7 @@ void RDMAConnectedSocketImpl::notify()
   int ret;
 
   ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  assert(ret == sizeof(i));
```

Comment:
@joscollin  Thanks for comments. since function returns void no point in taking ret also.
Can we make this way?
assert(sizeof(i) == write(notify_fd, &i, sizeof(i)));

write() Returns the number of bytes that were written. If value is negative, then the system call returned an error.
assert(exp) if expression evaluates to FALSE, displays an error message aborts.